src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,164 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs: (
+    font-size var(--pf-global--FontSize--xs)
+  ),
+  font-size-sm: (
+    font-size var(--pf-global--FontSize--sm)
+  ),
+  font-size-md: (
+    font-size var(--pf-global--FontSize--md)
+  ),
+  font-size-lg: (
+    font-size var(--pf-global--FontSize--lg)
+  ),
+  font-size-xl: (
+    font-size var(--pf-global--FontSize--xl)
+  ),
+  font-size-2xl: (
+    font-size var(--pf-global--FontSize--2xl)
+  ),
+  font-size-3xl: (
+    font-size var(--pf-global--FontSize--3xl)
+  ),
+  font-size-4xl: (
+    font-size var(--pf-global--FontSize--4xl)
+  )
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light: (
+    font-weight var(--pf-global--FontWeight--light)
+  ),
+  font-weight-normal: (
+    font-weight var(--pf-global--FontWeight--normal)
+  ),
+  font-weight-bold: (
+    font-weight var(--pf-global--FontWeight--bold)
+  )
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+// Font color options
+$pf-u-font-color-options: (
+  color-100: (
+    color var(--pf-global--Color--100)
+  ),
+  color-200: (
+    color var(--pf-global--Color--200)
+  ),
+  color-300: (
+    color var(--pf-global--Color--300)
+  ),
+  color-400: (
+    color var(--pf-global--Color--400)
+  ),
+  icon-color-light: (
```

Comment:
there are a lot more colors here than are in the docs. Can you make sure the docs are updated to reflect all of the classes?